openhtf/io/proto/guzzle.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
for the purposes of OpenHTF, guzzle is the only context in which a 'test run proto' makes sense, so is this worth pulling out into a separate file?  i don't feel too strongly about it, but just a suggestion, as i don't ever see using test runs protos without using the guzzle protos (which are ~10 additional lines, you have more added lines in copyright header than content, heh).